ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      delaySeconds: 35,
+      response: Responses.getUnder(),
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death IV',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CAF', capture: false }),
+      // Circles appear at the end of the cast.
+      delaySeconds: 4,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid Growing Circles',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death IV Followup',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CAF', capture: false }),
+      delaySeconds: 12,
+      // TODO: or "Avoid Growing Circles (again lol)"?
+      response: Responses.moveAway(),
+    },
+    {
+      id: 'Zadnor Diablo Aetheric Boom Raidwide',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CB3', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Diablo Aetheric Boom Balloons',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CB3', capture: false }),
+      // Don't warn people to preposition here, because they probably need
+      // heals after the initial hit before popping these.
+      delaySeconds: 5.5,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Pop Balloons',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Deadly Dealing',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CC2' }),
+      // TODO: these feel really late with 5 seconds, should they call instantly at 7?
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      alertText: (data, _matches, output) => {
+        // TODO: how does this loop?
+        if (data.diabloSeenDealing)
+          return output.knockbackNox();
+        return output.knockbackBits();
+      },
+      run: (data) => data.diabloSeenDealing = true,
+      outputStrings: {
+        knockbackBits: {
+          en: 'Knockback (away from bits)',
+        },
+        knockbackNox: {
+          en: 'Knockback (into empty corner)',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Void Systems Overload',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CB7', capture: false }),
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'Zadnor Diablo Pillar Of Shamash Spread',
+      // 5CBC damage
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: (data) => data.ce === 'dalriadaDiablo',
+      preRun: (data, matches) => {
+        data.diabloPillar = data.diabloPillar || [];
+        data.diabloPillar.push(matches.target);
+      },
+      alertText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.text();
+      },
+      outputStrings: {
+        text: {
+          en: 'Laser Spread on YOU',
```

Comment:
Is 'Laser Spread' a critical distinction from a regular 'Spread'?

Reply:
Maybe "Laser on YOU"? I guess I think of "spread" as being a circular aoe where you get away from others and a "laser" as being a "don't point this at other people".